java/src/org/openqa/selenium/edge/EdgeDriver.java

```diff
@@ -62,8 +68,12 @@ public static RemoteWebDriverBuilder builder() {
   }
 
   private static class EdgeDriverCommandExecutor extends ChromiumDriverCommandExecutor {
-    public EdgeDriverCommandExecutor(DriverService service) {
-      super(service, getExtraCommands());
+    public EdgeDriverCommandExecutor(DriverService service) throws URISyntaxException {
+      this(service, ClientConfig.defaultConfig());
+    }
+
+    public EdgeDriverCommandExecutor(DriverService service, ClientConfig clientConfig) throws URISyntaxException {
```

Comment:
Please remove the `throws clause` here and also kindly help clean up the same from all the constructors as well. It's not needed.